test/actions/katello/host/attach_subscriptions_test.rb

```diff
@@ -33,5 +33,22 @@ class AttachSubscriptionsTest < ActiveSupport::TestCase
                                           :quantity => 2, :pool_uuid => @pool.cp_id
       end
     end
+
+    describe 'attach subscriptions failure' do
+      let(:action_class) { ::Actions::Katello::Host::AttachSubscriptions }
+
+      it 'plans' do
+        action = create_action action_class
+        action.expects(:action_subject).with(@host)
+
+        @host.expects(:subscription_facet).returns(nil)
+
+        pools_with_quantities = [::Katello::PoolWithQuantities.new(@pool, [1, 2])]
```

Comment:
For this test I don't think the pools_with_quantities comes into play at all since we error out earlier. IMO we can omit this